modules/iam-read-only-policy/variables.tf

```
@@ -0,0 +1,58 @@
+variable "create_policy" {
+  description = "Whether to create the IAM policy"
+  type        = bool
+  default     = true
+}
+
+variable "name" {
+  description = "The name of the policy"
+  type        = string
+  default     = ""
+}
+
+variable "path" {
+  description = "The path of the policy in IAM"
+  type        = string
+  default     = "/"
+}
+
+variable "description" {
+  description = "The description of the policy"
+  type        = string
+  default     = "IAM Policy"
+}
+
+variable "allowed_services" {
+  description = "List of services to allow Get/List/Describe/View options. Service name should be the same as corresponding service IAM prefix. See what it is for each service here https://docs.aws.amazon.com/service-authorization/latest/reference/reference_policies_actions-resources-contextkeys.html"
+  type        = list(string)
+}
+
+variable "additional_policy" {
```

```suggestion
variable "additional_policy_json" {
```

This is how such variable is called in other modules (e.g. Lambda, Step Functions).